src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const smallTerm = 3;
+  const smallTermDiscount = 20;
+  const valueDay = 40;
+
+  if (days >= longTerm) {
+    return days * valueDay - longTermDiscount;
```

Comment:
The discount logic here applies a fixed discount of 50 for rentals of 7 days or more. Ensure this aligns with the task requirements. If the discount should be applied per day after a certain threshold, the logic needs to be adjusted.